main/src/cgeo/geocaching/connector/AbstractLogin.java

```diff
@@ -94,4 +96,9 @@ public StatusCode login(@Nullable final Credentials credentials) {
     @NonNull
     protected abstract StatusCode login(final boolean retry, @NonNull final Credentials credentials);
 
+    public void increaseActualCachesFound() {
+        if (getActualCachesFound() >= 0) {
```

Comment:
Introduce a local variable for getActualCachesFound()?

Reply:
I could also reference the field directly

Reply:
Yes, that sounds like a good idea.